client/signup/steps/about/index.jsx

```diff
@@ -106,17 +106,20 @@ class AboutStep extends Component {
 
 	setPressableStore = ref => ( this.pressableStore = ref );
 
-	onSiteTopicChange = ( { vertical_id, vertical_name, vertical_slug } ) => {
+	onSiteTopicChange = ( { parent, verticalId, verticalName, verticalSlug } ) => {
 		this.setState( {
-			verticalId: vertical_id,
-			siteTopicValue: vertical_name,
-			siteTopicSlug: vertical_slug,
+			verticalId: verticalId,
+			siteTopicValue: verticalName,
+			siteTopicSlug: verticalSlug,
```

Comment:
Should we be adding `parentId` to the state to persist it in the data layer like we do [in the site-topic step](https://github.com/Automattic/wp-calypso/blob/master/client/signup/steps/site-topic/index.jsx#L72)?

Reply:
@michaeldcain  We probably should. 👍 Thanks for thinking of that! This PR was a bit stale, so good chance for a rebase too.